share_app/src/main/java/org/odk/share/tasks/UploadJob.java

```diff
@@ -340,6 +340,9 @@ private void sendBlankForm(Cursor cursor) {
                 sbResult.append(getContext().getString(R.string.id, formId) + " " +
                         getContext().getString(R.string.success, getContext().getString(R.string.blank_form_count,
                                 getContext().getString(R.string.sent))));
+            } else {
```

Comment:
Why added this change?

Reply:
@lakshyagupta21 Sir, This resolves the empty dialog box error, issue number #157